src/test/kotlin/com/music/awesomemusic/controllers/AccountControllerTest.kt

```diff
@@ -40,66 +45,81 @@ class AccountControllerTest {
     @Autowired
     private lateinit var tokenRepository: ITokenRepository
 
-    lateinit var account: AwesomeAccount
-
-    @Before
-    fun prepareAccount() {
-        account = accountService.createAccount(AccountSignUpForm("holker", "123456", "test.mail@gmail.com", false))
-    }
-
     @After
     fun deleteAccount() {
         tokenRepository.deleteAll()
         accountRepository.deleteAll()
     }
 
+
     @Test
     @Throws(Exception::class)
-    fun signInShouldReturnOk() {
+    fun shouldMeIfAccountExistsExpected200() {
+
+        val account = accountService.createAccount(AccountSignUpForm("test5", "1234", "emailTest5", false))
+        val authorities = arrayListOf<String>()
+
+        val token = tokenProvider.createToken(account.username, authorities)
+
         val headers = HttpHeaders()
         headers.contentType = MediaType.APPLICATION_JSON
-        val request = HttpEntity<String>("{ \"login\" : \"holker\", \"password\" : \"123456\" }", headers)
-        val response = restTemplate.postForEntity("http://localhost:$port/api/user/sign-in", request,
-                String::class.java)
-        assertEquals(response.statusCode, HttpStatus.OK)
+
+        headers.set("Authorization", "AwesomeToken $token")
+
+
+        val request = HttpEntity<String>(headers)
+        val responseEntity = restTemplate.exchange("http://localhost:$port/api/user/me", HttpMethod.GET, request, String::class.java)
+
+        assertEquals(responseEntity.statusCode, HttpStatus.OK)
+
     }
 
     @Test
-    fun meShouldReturnOk() {
-        val token = tokenProvider.createToken(account.username, emptyList<String>())
+    @Throws(Exception::class)
+    fun shouldMeIfAccountNoExistsExpected403() {
```

Comment:
What means "should me"  ?
Doesn't quite fit naming convention https://boring-team.atlassian.net/wiki/spaces/AWESOME/pages/135102465/Test+Convention
```suggestion
    fun shouldReturn403IfAccountDoesntExist() {
```